frontend/app/index/reports/controller.js

```diff
@@ -96,6 +97,7 @@ export default class IndexReportController extends Controller {
 
   @action
   async createEmptyReport() {
+    if (!this.checkForEmptyRecord) return;
```

Comment:
without this we run into a race condition and the magic-link record and an empty record will be created

![image](https://github.com/user-attachments/assets/3410d3c1-3c18-4492-8c33-b2803d4a42b0)

Reply:
Thanks for the hint! Now I get it ;-)